pkg/api/types.go

```diff
@@ -1296,6 +1296,7 @@ const (
 	ClusterProfileOpenStackVexxhost     ClusterProfile = "openstack-vexxhost"
 	ClusterProfileOpenStackPpc64le      ClusterProfile = "openstack-ppc64le"
 	ClusterProfileOpenStackOpVexxhost   ClusterProfile = "openstack-operators-vexxhost"
+	ClusterProfileOpenStackNercDev      ClusterProfile = "openstack-nerc-dev"
```

Comment:
why do you call it `openstack-nerc-dev` btw, since it's for CI? Could we just name is `ClusterProfileOpenStackNerc` maybe? to avoid confusion and complexity.

Reply:
We have two tenants on NERC and the one called CI… is the other one 😬 on which we don’t remotely have quota. We are setting up jobs on this tenant because @MaysaMacedo identified it as the one tenant where we happen to be able to run tests for  couple iterations of CAPI-installer (that is: only one external network).

I understand that the naming is suboptimal but at this stage I think it’s the one more likely to preserve our sanity

Reply:
meh